flask/tests/test_samples.py

```diff
@@ -136,3 +136,78 @@ def test_create_tissue_sample(test_database, client, login_as):
     assert len(sample.datasets) == 0
     assert sample.created_by == 1
     assert sample.updated_by == 1
+
+
+# PATCH /api/tissue_samples
+
+
+def test_update_tissue_sample_admin(client, test_database, login_as):
+    """
+    PATCH /api/datasets/:id as an administrator
+    """
+    login_as("admin")
+
+    assert client.patch("/api/tissue_samples/2").status_code == 415
+    # Nonexistent
+    assert (
+        client.patch("/api/tissue_samples/400", json={"foo": "bar"}).status_code == 404
+    )
+    # Assume user identity that does not have permission
+    assert (
+        client.patch("/api/tissue_samples/2?user=1", json={"foo": "bar"}).status_code
+        == 404
+    )
+
+    # Bad tissue_sample_type
+    assert (
+        client.patch(
+            "/api/tissue_samples/2", json={"tissue_sample_type": "foo"}
+        ).status_code
+        == 400
+    )
+
```

Comment:
minor - your code is pretty self explanatory but a comment to explain what is being tested would be good to have here